docs/source/datasets.rst

```diff
@@ -75,16 +75,20 @@ SSVEP Datasets
     Lee2019_SSVEP
 
 
+
 --------------
-c-VEP Datasets
+c-VEP and Burst-VEP Datasets
```

Comment:
```suggestion
c-VEP Datasets
```

Reply:
Same comment as previous but we an adapt either it is a Burst-VEP or a m sequence